go.mod

```diff
@@ -5,11 +5,12 @@ go 1.13
 require (
 	github.com/aquasecurity/fanal v0.0.0-20200306122936-f0a17242a9a0
 	github.com/aquasecurity/go-dep-parser v0.0.0-20190819075924-ea223f0ef24b
-	github.com/aquasecurity/trivy-db v0.0.0-20191226181755-d6cabf5bc5d1
+	github.com/aquasecurity/trivy-db v0.0.0-20200314021825-6460fab71aaa
```

Comment:
Note to self: Change this to master once https://github.com/aquasecurity/trivy-db/pull/34 is merged.